### DISCUSSION TITLE
Wiki: exemplo baixar notas fornecedores indisponível

### DISCUSSION THREAD

**Discussion by agrings** [TadaSoftware/PyNFe#223] [Q&A] [answered]
Olá,

No wiki, na página Consulta-Distribuição.md, há um link "Exemplo de como baixar todas as notas de fornecedores".
Mas a página alvo parece não parece estar mais disponível: https://github.com/TadaSoftware/PyNFe/wiki/Consulta-Distribuicao-1
Qual a razão?

**Comment by leogregianin** [comment, marked as answer]
@agrings corrigido o link

**Reply by agrings** [reply]
Perfeito. Obrigado @leogregianin
